src/pages/iou/request/IOURequestStartPage.tsx

```diff
@@ -75,8 +75,7 @@ function IOURequestStartPage({
 
     const isExpenseChat = ReportUtils.isPolicyExpenseChat(report);
     const isExpenseReport = ReportUtils.isExpenseReport(report);
-    const shouldDisplayDistanceRequest =
-        !!canUseCombinedTrackSubmit || !!canUseP2PDistanceRequests || isExpenseChat || isExpenseReport || (isFromGlobalCreate && iouType !== CONST.IOU.TYPE.SPLIT);
+    const shouldDisplayDistanceRequest = !!canUseCombinedTrackSubmit || isExpenseChat || isExpenseReport || (isFromGlobalCreate && iouType !== CONST.IOU.TYPE.SPLIT);
```

Comment:
I think `shouldDisplayDistanceRequest` should always be true

Reply:
@neil-marcellini @robertjchen If possible, please take a look at this comment

Reply:
Yeah that's actually a good point, I'll remove the condition entirely.